packaging/hudi-cli-bundle/pom.xml

```diff
@@ -0,0 +1,243 @@
+                  <include>com.fasterxml:classmate</include>
+                  <include>com.fasterxml.woodstox:woodstox-core</include>
+                  <include>com.google.code.gson:gson</include>
+                  <include>com.google.re2j:re2j</include>
+                  <include>com.jakewharton.fliptables:fliptables</include>
+
+                  <include>jakarta.el:jakarta.el-api</include>
+                  <include>jakarta.validation:jakarta.validation-api</include>
+                  <include>net.java.dev.jna:jna</include>
+
+                  <include>org.apache.httpcomponents:httpclient</include>
+                  <include>org.apache.httpcomponents:httpcore</include>
+                  <include>org.apache.httpcomponents:fluent-hc</include>
+                  <include>org.codehaus.woodstox:stax2-api</include>
+                  <include>org.fusesource.jansi:jansi</include>
+                  <include>org.glassfish:jakarta.el</include>
+                  <include>org.hibernate.validator:hibernate-validator</include>
+                  <include>org.jboss.logging:jboss-logging</include>
+                  <include>org.jline:*</include>
+                  <include>org.springframework*:*</include>
+                  <include>org.springframework.boot:*</include>
+                  <include>org.yaml:*</include>
+                </includes>
+              </artifactSet>
+              <relocations>
+                <relocation>
+                  <pattern>com.google.code.gson.</pattern>
+                  <shadedPattern>org.apache.hudi.com.google.code.gson.</shadedPattern>
+                </relocation>
+              </relocations>
+              <filters>
+                <filter>
+                  <artifact>*:*</artifact>
+                  <excludes>
+                    <exclude>META-INF/*.SF</exclude>
+                    <exclude>META-INF/*.DSA</exclude>
+                    <exclude>META-INF/*.RSA</exclude>
+                    <exclude>**/*.proto</exclude>
+                    <exclude>**/Log4j2Plugins.dat</exclude>
+                  </excludes>
+                </filter>
+              </filters>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+    <resources>
+      <resource>
+        <directory>hudi-cli/src/main/resources</directory>
+      </resource>
+    </resources>
+  </build>
+
+  <dependencies>
+    <!-- Hoodie -->
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-cli</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-utilities_${scala.binary.version}</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>${hadoop.version}</version>
+      <scope>compile</scope>
```

Comment:
nit: we can just use the default scope, same for the other dependencies

Reply:
Actually we need this as `compile` since the root pom currently has this as `provided`.